app/src/main/java/com/hamdan/acceleromatorminiproject/viewmodels/SensorViewModel.kt

```diff
@@ -55,6 +88,7 @@ class SensorViewModel : ViewModel() {
     fun checkAcceleration(acceleration: Acceleration, onFail: () -> Unit) {
         sensorState.asCountingOrNull()?.let {
```

Comment:
feel free to replace these with the is format if you prefer.

Reply:
Going to leave them as I think making those changes would bloat the PR for this feature